src/parcel_snoopi.jl

```diff
@@ -349,11 +349,26 @@ const default_blacklist = Set([
 ])
 
 """
-Removes everything that can't eval.
+    exhaustive_remover!(pcstatements, modul::Module)
```

Comment:
Note that changing it so that `can_eval` doesn't run by default is technically a breaking change. I don't want to have to go to version 2.0.0 just because of that, please change back.

Reply:
It is not the default behavior. It is set to `false`
https://github.com/aminya/SnoopCompile.jl/blob/9b5387f5a314bce1e2e3b4a57e77f8305b2d0235/src/parcel_snoopi.jl#L200

Reply:
Right, but the current version of the package always checks that statements can be evaluated. Every addition is passed through `add_if_evals!`, and there is no option to turn it off.

Reply:
Oh, I did not see the `can_eval`! If this is the default behavior, probably we should set an option to allow user to set it off. The default can be `true` to avoid breakage.

Also, we should merge `exhaustive_remover` code with `add_if_evals`. This can definitely improve the performance.